Source/WebKit/WebProcess/InjectedBundle/API/mac/WKWebProcessPlugInBrowserContextController.mm

```diff
@@ -546,15 +546,15 @@ bool shouldNotifyOnFormChanges(WebKit::WebPage*) final
         {
             auto controller = m_controller.get();
             if (!controller)
-                return false;
+                return true;
```

Comment:
Why are we changing these return values?

Reply:
Oh I guess you want to make Document::didAssociateFormControl schedule a timer.
I think we should add a new check for document.hasEventListenersOfType instead.